pytest_mitmproxy.py

```diff
@@ -1 +1,64 @@
 # -*- coding: utf-8 -*-
+import re
+import os
+import shutil
+import pytest
+import urllib
+from contextlib import contextmanager
+from subprocess import Popen, SubprocessError, CalledProcessError
+from tempfile import TemporaryFile
+from urllib.parse import urlparse
+
+# clean the dumps each run
+if os.path.isdir("traffic_dumps"):
```

Comment:
Well as before: QA-395. I'm coasting on the README saying this isn't usable for anybody else yet to get away with the quick/easy/not-really-needing-tests solutions until after the more pressing issues for which the hardcoding is sufficient.